examples/alb-test/tester.tf

```diff
@@ -0,0 +1,64 @@
+module "vpc" {
+  source              = "../../modules/vpc-scenario-2"
+  cidr                = "192.168.0.0/16"
+  region              = "us-east-2"
+  name_prefix         = "alb_test"
+  public_subnet_cidrs = ["192.168.1.0/24", "192.168.2.0/24", "192.168.3.0/24"]
+  azs                 = ["us-east-2a", "us-east-2b", "us-east-2c"]
+}
+
+module "asg" {
+  source               = "../../modules/asg"
+  ami                  = "ami-0efd7a9e366418dda"
+  key_name             = "shida-east-2"
+  azs                  = ["us-east-2a", "us-east-2b", "us-east-2c"]
+  max_nodes            = 3
+  min_nodes            = 3
+  subnet_ids           = module.vpc.public_subnet_ids
+  name_prefix          = "alb_test"
+  security_group_ids   = [aws_security_group.wide_open.id]
+  lb_target_group_arns = module.alb.target_group_arns
+}
+
+resource "aws_security_group" "wide_open" {
+  vpc_id = module.vpc.vpc_id
+
+  ingress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+module "alb" {
+  source        = "../../modules/alb"
+  ports         = ["80"]
+  vpc_id        = module.vpc.vpc_id
+  sg_ids        = [aws_security_group.wide_open.id]
+  identifier    = "albest"
```

Comment:
does this identifier have any special meaning? I guess the first part is ALB but what does suffix mean? Shouldn't it be "alb_test" as above?

Reply:
Ye.... There is a 6 char limit....

Reply:
Maybe "albtst"? Sounds a bit more readable, BTW where does the limit come from?

Reply:
OK. For the limitation, I guess it is something missed in TF doc. For example, target group, says clearly that name_prefix has 6 char limitation. But for lb, it says nothing, but when planning, it throws error.